tensorflow/examples/label_image/label_image.py

```diff
@@ -0,0 +1,125 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
```

Comment:
I believe Pete meant, add a comment in the file that explains how to use it. I understand you have it in the readme, but having it in the file is much more likely to be found.

Also, I do believe your description in the readme is incorrect -- this file will not be installed with the pip package as it is today, I believe. @petewarden I may be wrong about that though.